src/main/java/pl/marekjava/App.java

```diff
@@ -45,9 +46,9 @@ private static void playGame(List<Player> players) {
                     System.out.println(pm.getObscuredPassword());
                     String input = scan.nextLine();
                     if (input.length() == 1) {
-                        guessLetter(password, input);
+                        guessLetter(password, input, players.get(j));
```

Comment:
Myślę, że w tym momencie warto już tego fora 'klasycznego' przerobić dla czytelności na 'enhanced for' czyli zamiast
`for (int j = 0; j < players.size(); j++)` zrobić `for(Player player : players)` i potem zamiast ` .get(j)` po prostu odnosić się do `player` 

Będzie zdecydowanie czytelniej, bo samego `j` chyba nigdzie nie wykorzystujemy

Reply:
Poprawione